src/catalog/src/durable/impls/persist.rs

```diff
@@ -741,7 +744,6 @@ impl ReadOnlyDurableCatalogState for PersistCatalogState {
     #[tracing::instrument(level = "debug", skip(self))]
     async fn expire(self: Box<Self>) {
         self.write_handle.expire().await;
-        self.listen.expire().await;
```

Comment:
As far as I remember, expiring/graceful cleanup was best effort, so I'm not too concerned about removing this. But are there any other impacts? If so can you leave a comment?

Reply:
I don't think there are any impacts, I'll add a comment. Once/if this merges: https://github.com/MaterializeInc/materialize/pull/23462 we can bring back the cleanup.

Reply:
Added.